api/handlers_images.go

```diff
@@ -144,3 +144,48 @@ func (s *server) ImageUpdateHandler(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(http.StatusNoContent)
 }
+
+func (s *server) ImageCreateHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := s.mapAccountNumber(vars["account"])
+
+	req := &Ec2ImageCreateRequest{}
+	if err := json.NewDecoder(r.Body).Decode(req); err != nil {
+		msg := fmt.Sprintf("cannot decode body into update image input: %s", err)
+		handleError(w, apierror.New(apierror.ErrBadRequest, msg, err))
+		return
+	}
+
+	if req.InstanceId == nil || req.Name == nil || req.CopyTags == nil || req.ForceReboot == nil {
+		handleError(w, apierror.New(apierror.ErrBadRequest, "missing required fields: instance_id, name, copy_tags, force_reboot", nil))
+		return
+	}
+
+	policy, err := generatePolicy([]string{"ec2:CreateImage"})
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+
+	orch, err := s.newEc2Orchestrator(r.Context(), &sessionParams{
+		role:         fmt.Sprintf("arn:aws:iam::%s:role/%s", account, s.session.RoleName),
+		inlinePolicy: policy,
+		policyArns: []string{
+			"arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess",
+		},
+	})
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+
+	imageId, err := orch.createImage(r.Context(), req)
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+
+	w.WriteHeader(http.StatusCreated)
+	w.Write([]byte(imageId))
```

Comment:
```
handleResponseOk(w, out)
```

Reply:
I thought about it but `handleResponseOk` doesn't send the status 201. Please let me know your thoughts.

Reply:
We don't really need the 201, let's standardize on 200 and use the `handleResponseOk`